### ISSUE TITLE
dirrefent simple layout 

### ISSUE THREAD

**Issue by bchtrue** [perfect-things/perfect-home#143]
He again :-)

Is it possible to add this simple layout as an option? Maybe it will be useful. At least look clean. 
https://github.com/Farow/startpage 

Only layout + all other features from your plugin.

Or maybe better if it will be possible to use "merged mode": 
Current layout with big icons and this simple links layout with names for some links at the bottom after it.

**Comment by tborychowski** [comment]
have you tried docked folders?
![](https://raw.githubusercontent.com/perfect-things/perfect-home/master/_stuff/5-docked-folders.png)

with a little bit of css you could style them the same as they look on this startpage (let me know if you need help with this).


### As for the request:
Considering that your request is the second of this kind (considering there are ~600 firefox + ~100 chrome users). The second one was similar (layout) but different (desktop icons-like, free-hand placement).
And catering for all possible requests like these make things unnecessary complex to develop & test.

I will leave this issue here and only if there are more requests like this - I will consider implementing it :-)
(I hope that's fair enough for you?)

**Comment by bchtrue** [comment]
@tborychowski Thank you, just wanted to be helpful and show not a bad option as layout. Didn't mean to sound like I'm demanding something. It's just a suggestion)

**Comment by tborychowski** [comment]
@bchtrue no, no, your feature request was perfectly fine and it is appreciated! :-)

It's just that when you develop an app that has no marketing, no analytics, (and no funds), it's sometimes difficult to weight the features and bugs against the resources you have. 
And making everyone happy is impossible (even with all the resources in the world) and I find it difficult to say "no" to requests such as yours, but I have to, and hence - my attempt to explain myself :-)
